lib/services/argument-handler.js

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = argumentHandlerServiceFactory;
```

Comment:
@yyscamper This looks like a new service.  Where does the ArgumentHandlerService get used?